Exiled.API/Features/Player.cs

```diff
@@ -123,7 +123,7 @@ public Player(GameObject gameObject)
         /// <summary>
         /// Gets a <see cref="Dictionary{TKey, TValue}"/> containing all <see cref="Player"/>'s on the server.
         /// </summary>
-        public static Dictionary<GameObject, Player> Dictionary { get; } = new(20);
+        public static Dictionary<GameObject, Player> Dictionary { get; } = new(20, new ReferenceHub.GameObjectComparer());
```

Comment:
As I remember, @Undid-Iridium Told me about storing gameObjects in memory, and he was right, the usual check (at least for null) does not work with gameObjects, and I saw that NWs use this TypeComparer in their dictionary, so I think it's better